src/collection-browser.ts

```diff
@@ -245,70 +250,111 @@ export class CollectionBrowser
   }
 
   render() {
+    this.doesHaveSearchTermOrResult();
```

Comment:
Suggestion: 
- better name for this method.
- show intent that is hidden in method.

Example: `this.emptyPlaceholder = setPlaceholder()`